feeds/wlan-ap/opensync/src/platform/openwrt/src/lib/target/src/uci.c

```diff
@@ -162,12 +162,15 @@ int blob_to_uci_section(struct uci_context *uci,
 	ret = uci_lookup_ptr(uci, &p, tuple, true);
 	if (ret != UCI_OK)
 		goto err_out;
+	if (!a && !(p.flags | UCI_LOOKUP_COMPLETE))
```

Comment:
Why was this needed?

Reply:
The next line will create a section if none exists.  This line prevents a caller who only wanted to delete options from a section from creating an empty section.  For instance, this may occur if Wifi_RRM_Config is removed for a radio and a Wifi_VIF_Config is removed at the same time.  Depending on the timing,  Wifi_RRM_Config may try to delete the RRM config from a no longer existent VIF.  Without this check, we may end up with a partial wifi-iface section in wireless and confuse hostApd.

Reply:
Got it. Thanks!